config/manager/manager.yaml

```diff
@@ -24,13 +24,40 @@ spec:
     spec:
       securityContext:
         runAsNonRoot: true
+      volumes:
+        - name: config
+          configMap:
+            name: dspo-config
       containers:
       - command:
         - /manager
         args:
         - --leader-elect
+        - --config
+        - /home/config
         image: controller:latest
         name: manager
+        env:
+          - name: IMAGES_APISERVER
+            value: quay.io/modh/odh-ml-pipelines-api-server-container:v1.18.0-8
+          - name: IMAGES_ARTIFACT
+            value: quay.io/modh/odh-ml-pipelines-artifact-manager-container:v1.18.0-8
+          - name: IMAGES_PERSISTENTAGENT
+            value: quay.io/modh/odh-ml-pipelines-persistenceagent-container:v1.18.0-8
+          - name: IMAGES_SCHEDULEDWORKFLOW
+            value: quay.io/modh/odh-ml-pipelines-scheduledworkflow-container:v1.18.0-8
+          - name: IMAGES_VIEWERCRD
+            value: quay.io/modh/odh-ml-pipelines-viewercontroller-container:v1.18.0-8
+          - name: IMAGES_CACHE
+            value: registry.access.redhat.com/ubi8/ubi-minimal
+          - name: IMAGES_MOVERESULTSIMAGE
+            value: busybox
```

Comment:
see comment above

Reply:
this is not scope in this pr, addressed in this PR: https://github.com/opendatahub-io/data-science-pipelines-operator/commit/32bb5f78cd24dcb51e63c370c6ad054413446f32